### PR TITLE
Workflows: update versions

### PR DESCRIPTION
**Pull request by gcabiddu**
Update versions to latest available for all workflows in QATlib.

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -25,13 +25,11 @@ jobs:
...
     - name: Checkout repository
-      # uses: actions/checkout@v3
-      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      # uses: github/codeql-action/init@v2
-      uses: github/codeql-action/init@c3b6fce4ee2ca25bc1066aa3bf73962fda0e8898
+      uses: github/codeql-action/init@b8efe4dc6ab6d31abe3ec159420d2a4916880800
```
Comment (fionatrahe) [inline .github/workflows/codeql.yml:32]: Trying to understand why you picked those specific commit ids. It's in between these two tags. Should we not pick the latest tagged version?
https://github.com/github/codeql-action/commits/v3.26.5
https://github.com/github/codeql-action/commits/v3.26.6

**Comment by mythi** [comment]
> Thanks @mythi. Should we take https://github.com/intel/intel-device-plugins-for-kubernetes/ as a reference?

That should work. All our action versions (e.g., the CodeQL versions) are up-to-date and something dependabot knows how to update.

File: .github/workflows/codeql.yml [hunk 2]
```diff
@@ -49,7 +47,6 @@ jobs:
       run: make -j
 
     - name: Perform CodeQL Analysis
-      # uses: github/codeql-action/analyze@v2
-      uses: github/codeql-action/analyze@c3b6fce4ee2ca25bc1066aa3bf73962fda0e8898
+      uses: github/codeql-action/analyze@b8efe4dc6ab6d31abe3ec159420d2a4916880800
```
Comment (fionatrahe) [inline .github/workflows/codeql.yml:50]: same here. use v3.26.6

**Review by fionatrahe — APPROVED** [review]
Marking all comments as resolved as they're minor and will be fixed in a follow up PR in the next few days. Approving this for merge, to check if it fixes the OSSF error.

**Pull request merged**